tests/bpf2c_tests/expected/bindmonitor_sys.txt

```diff
@@ -1,3 +1,6 @@
+// Do not alter this generated file.
+// This file was generated from bindmonitor.o
+
 // Copyright (c) Microsoft Corporation
 // SPDX-License-Identifier: MIT
```

Comment:
My opinion is that the copyright and license id should come first so that the license checker script is happy:

```suggestion
// Copyright (c) Microsoft Corporation
// SPDX-License-Identifier: MIT

// Do not alter this generated file.
// This file was generated from bindmonitor.o
```